cmd/component.go

```diff
@@ -68,6 +71,36 @@ var componentSetCmd = &cobra.Command{
 	},
 }
 
+var componentDescribeCmd = &cobra.Command{
+	Use:   "describe",
+	Short: "describe the given application",
+	Run: func(cmd *cobra.Command, args []string) {
+		client := getOcClient()
+		er := args[0]
```

Comment:
`er` is not descriptive variable name.


```
▶ ./ocdev component describe
panic: runtime error: index out of range
```
The command should have one optional argument (component name).
If that argument is not provided that default component should be used.

```
#describe current component 
ocdev describe 

# describe "frontend" component"
ocdev decribe frontend
```